aunt-leahs-app/src/components/locationTable/locationTable.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+
+class LocationSelection extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            locations: [
+                {
+                    name: "Tree Lot",
+                    id: 1,
+                    address: "1234 East Mall"
+                },
+                {
+                    name: "Thrift Store",
+                    id: 2,
+                    address: "1234 Robson St"
+                },
+                {
+                    name: "Fundraising Event",
+                    id: 3,
+                    address: "1234 Wesbrook Mall"
+                }]
+        };
+
+        this.selectLocation = this.selectLocation.bind(this);
+
+    }
+
+    selectLocation(id){
+        var updatedLocations = this.state.locations.slice(0);
+        updatedLocations.forEach(l => {
+            if(l.id === id) {
+                l.selected = true;
+                this.props.onLocationSelect(l);
+            } 
+            else {
+                l.selected = false;
+            }
+        });
+
+        this.setState({locations: updatedLocations});
+    }
+
+    render() {
+        return <Table aria-label="simple table">
+            <TableBody>
+            {this.state.locations.map(location => (
+                <TableRow key={location.id} 
+                    onClick={(e) => this.selectLocation(location.id)}
+                    hover={true}
```

Comment:
For boolean props that are always true, you should be able to simply write the name of the prop, as below.
```suggestion
                    hover
```

Reply:
I don't know what happened but I didn't modify any codes besides shiftDataPage.
So I don't know if I should make those changes.

Reply:
This has been taken cared of in another issue and is not part of this PR.